CONTRIBUTING.md

```diff
@@ -226,4 +226,4 @@ Accord Project documentation files are made available under the [Creative Common
 [clacorp]: http://code.google.com/legal/corporate-cla-v1.0.html
 
 [apache]: https://github.com/accordproject/ergo/blob/master/LICENSE
-[creativecommons]: http://creativecommons.org/licenses/by/4.0/
\ No newline at end of file
+[creativecommons]: http://creativecommons.org/licenses/by/4.0/
```

Comment:
This still needs to be removed.

Reply:
Hi @irmerk I didn't make any changes in this but it is showing it has changed in all changes but not reflecting any changes